doc/devel/documenting_mpl.rst

```diff
@@ -814,7 +814,9 @@ from having an plot generated insert "sgskip" somewhere in the filename.
 
 The format of these files is relatively straightforward.  Properly
 formatted comment blocks are treated as ReST_ text, the code is
-displayed, and figures are put into the built page.
+displayed, and figures are put into the built page.  Matplotlib uses the
+``# %%`` section separator so that IDEs will identify "code cells" to make
+it easy to re-run sub-sections of the example.
```

Comment:
There is an example of a tutorial below this which still has the long line.

Reply:
🤦🏻 I looked for our documentation on how to split sections and was a bit surprised we did not have any.  I am glad the problem was me and not our docs!  Thank you for catching this.